docs/csharp/quick-starts/arrays-and-collections.md

```diff
@@ -20,7 +20,7 @@ class.
 > [!NOTE]
 > If you're starting from the code you wrote in [dot.net](https://dot.net/), you already have the code written in this section. Jump to [Modify list contents](#modify-list-contents).
 
-This lesson assumes you've finished the online quick starts, and you've installed the [.NET Core SDK](http://dot.net/core) and [Visual Studio Code](https://code.visualstudio.com/). 
+This lesson assumes you've finished the online quick starts and you've installed the [.NET Core SDK](http://dot.net/core). You also need a code editor. If you don't have one, we recommend [Visual Studio](https://www.visualstudio.com/) for Mac or Windows. 
```

Comment:
Yep, this fixes the confusion 😄

Reply:
I wonder if we're making it sound like there is no option for Linux? Visualstudio.com is pretty responsive to the OS you are on, so I wonder if we could just say that Visual Studio has offerings for Windows, Mac, and Linux.

Reply:
Should I call out Visual Studio Code on Linux? I thought of that, but thought it might be too wordy. Thoughts?

Reply:
I just made an update here. Let me know what you think of the wording.

Reply:
Yep, I think that does the trick